tests/test_input_output.py

```diff
@@ -110,6 +110,132 @@ def test_write_desc_input_Nones(tmpdir_factory):
     assert no_gtol  # fail test if gtol line is written
 
 
+@pytest.mark.unit
+def test_descout_to_input(tmpdir_factory):
+    """
+    Test converting DESC output to a DESC input file.
+
+    To do that, we convert a DESC output file to a DESC input file
+    named desc_input_converted. We compare the boundary Fourier
+    coefficients of this file with the true values of the Fourier
+    coefficients given in desc_input_truth.
+    """
+    outfile_path = "./tests/inputs/LandremanPaul2022_QA_reactorScale_lowRes.h5"
+    tmpdir = tmpdir_factory.mktemp("desc_inputs")
+    tmp_path = tmpdir.join("input_LandremanPaul_QA")
+    tmpout_path = tmpdir.join("LandremanPaul2022_QA_reactorScale_lowRes.h5")
+    shutil.copyfile(outfile_path, tmpout_path)
+
+    ir = InputReader()
+    ir.descout_to_input(str(tmp_path), str(tmpout_path))
+    desc_input_truth = ".//tests//inputs//LandremanPaul2022_QA_reactorScale_lowRes"
+    desc_input_converted = tmp_path
+
+    with open(desc_input_truth) as f:
```

Comment:
instead of manually parsing both these files, its probably easier (and a better test of the code) to use `InputReader.parse_inputs` to get both as a dict of ndarray and then do regular array comparisons